google-cloud-clients/google-cloud-compute/README.md

```diff
@@ -12,7 +12,6 @@ Java idiomatic client for [Google Cloud Compute][cloud-compute].
 - [Product Documentation][compute-product-docs]
 - [Client Library Documentation][compute-client-lib-docs]
 
-> Note: This client is no longer receiving updates; new features in the Compute API will not be added to this client.
```

Comment:
The code samples in this file still use the deprecated code.

Reply:
Blocked by https://github.com/googleapis/gapic-generator/issues/2096

Reply:
Looks like this was resolved 😄